### ISSUE TITLE
Disable Ctrl+Q to not accidentally quit Rambox

### ISSUE THREAD

**Issue by kosli** [ramboxapp/community-edition#2291]
O365 web interface uses Ctrl+Q to mark mails as read. Sometimes this interferes with Rambox and it closes Rambox accidentally. It would be good if the keyboard shortcut(s) of Rambox would be configurable to prevent accidentally closing Rambox.

**Comment by maxerbox** [comment]
Every software should implement this shortcut.
It's very annoying to right click on the icon if you want to close definitely Rambox.
Just don't press it ^^.

**Comment by maurcs** [comment]
I'm a mac user. I would love to edit my pro app's keyboard shortcuts. This one in particular is disruptive to my usage of Rambox. In Gmail or chat I'm expecting to highlight and entire row of text by hitting CMD+SHIFT+LeftArrow, but instead i'm send to a whole new application. Same when hitting ALT+DEL it changes tabs instead of deleting an entire word.

> Navigation
> Windows/Linux: ALT + SHIFT +LeftArrow/RightArrow, CTRL + TAB, CTRL + SHIFT + TAB
> Mac: Cmd + SHIFT +LeftArrow/RightArrow, Cmd + TAB, Cmd + SHIFT + TAB

**Comment by pandruszkow** [comment]
Agreed. Ctrl+Q to quit an app is an extremely bad idea for long-lived apps like Rambox or web browsers, since it's very easy to hit by accident. Ctrl+Shift+Q (or nothing at all) as a preset would be much safer.

**Comment by gtrufitt** [comment]
Adding to @maurcs point about CMD+SHIFT+LeftArrow to highlight text - I'm close to leaving Rambox behind if this isn't possible to fix because it's an incredibly frustrating bug. 

Note CMD + OPT(alt) + Arrow is the standard (and used by Chrome for moving tabs).

https://support.google.com/chrome/answer/157179?hl=en-GB

**Comment by gitbobbed** [comment]
Chiming in to add that a second case of shortcut overlap. In WhatsApp, Control+1, Control+2 etc. switches between chats. Rambox hijacks that shortcut to switch between apps, which I would like to be able to avoid (by setting the Rambox shortcut to Control+Shift+1, Control+Shift+1 etc.)

**Comment by andresimi** [comment]
I twould be nice a shortcut to show / hide rambox!

**Comment by ThaoD5** [comment]
No update about alt ALT + DEL shortcut issue ? I might just drop using Rambox just because of that... It's really a pain...
